examples/generator-fullstack/.graphqlrc.yml

```diff
@@ -3,16 +3,12 @@ schema:
     require: ts-node/register
 documents: "./client/src/graphql/**/*.ts"
 extensions:
-  generate:
-    db:
-      dbConfig:
-        user: postgresql
-        password: postgres
-        database: users
-        host: localhost
-        port: 55432
-      database: pg
-    graphqlCRUD:
+  # Graphback configuration
+  graphback:
```

Comment:
@ardatan I have renamed generate to `graphback` as this will reflect configuration of the graphback generators that will be documneted on our website. We will have the same format everywhere. If we want to align with each cli command then we will probably need two keys now:

- Generate
- Migrate 

However, that will be more misleading IMHO